src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -545,6 +579,20 @@ void cpu_convert(const void *srcPtr,
         } else {
             cpu_memcpy(dstPtr, srcPtr, size * dstPrc.size());
         }
+    } else if (srcPrc == Precision::BIN) {
+        if (srcPrc.bitsSize() != 1)
+            IE_THROW() << "cpu_convert can't convert from: " << srcPrc << " (bitsSize == " << srcPrc.bitsSize()
+                << ") precision to: " << dstPrc << ". Not implemented.";
+        ConvertFromBinContext ctx = {
+                srcPtr,
+                dstPtr,
+                size,
+                false
+        };
+        OV_SWITCH(intel_cpu, ConvertFromBinPrecision, ctx, dstPrc, INTEL_CPU_CVT_FROM_BIN_LIST);
+        if (!ctx.converted)
+            IE_THROW() << "cpu_convert can't convert from: " << srcPrc << " (bitsSize == " << srcPrc.bitsSize()
```

Comment:
Please, use angle braces to better maintainability

Reply:
Ok, fixed.